app/controllers/Giraffe.scala

```diff
@@ -117,12 +117,16 @@ class Giraffe(paymentServices: PaymentServices) extends Controller {
       .withCookies(Test.createCookie(chosenVariants.v1), Test.createCookie(chosenVariants.v2))
   }
 
-
   def thanks(countryGroup: CountryGroup) = NoCacheAction { implicit request =>
-    val redirectUrl = routes.Giraffe.contribute(countryGroup).url
 
+    val title = countryGroup match {
+      case CountryGroup.Australia => "Thank you for supporting Guardian Australia"
```

Comment:
We should keep an eye on the number of branches we have like this - might end up modelled as a case class for specific branding.